pkg/rpcclient/rpc.go

```diff
@@ -1285,3 +1285,27 @@ func (c *Client) TerminateSession(sessionID uuid.UUID) (bool, error) {
 
 	return resp, nil
 }
+
+// GetRawNotaryTransaction returns a fallback or main transaction that was
+// previously added to the memory pool by P2PNotaryRequest.
+func (c *Client) GetRawNotaryTransaction(hash util.Uint256) (*transaction.Transaction, error) {
+	var (
+		params = []any{hash.StringLE(), 1} // 1 for verbose.
+		resp   = &transaction.Transaction{}
+		err    error
+	)
+	if err = c.performRequest("getrawnotarytransaction", params, resp); err != nil {
+		return nil, err
+	}
+	return resp, nil
+}
+
+// GetRawNotaryPool returns hashes from all the verified transactions,
```

Comment:
"... hashes of main P2PNotaryRequest transactions that are currently in the RPC node's notary request pool with corresponding hashes of fallback transactions."

Reply:
Fixed.